### ISSUE TITLE
Safari: Not able to navigate to office fabric buttons using keyboard

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#6580]
### Bug Report 
 
- __Package version(s)__: 6.74.1
- __Browser and OS versions__: Safari with macOS Sierra 10.12.6
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: Azure DevOps
 
#### Describe the issue:
 
In Safari browser, not able to tab to office fabric buttons using keyboard.
 
#### Actual behavior:
 
In Safari browser, not able to tab to office fabric buttons using keyboard.
 
#### Expected behavior:
 
It should be a tabstop

**Comment by Vitalius1** [comment]
@b-priyank does this happen in your app? I can confirm that this is happening on https://developer.microsoft.com/en-us/fabric#/components/button. In Safari it skips tab stops for Default and Compound buttons examples.

**Comment by b-priyank** [comment]
Yes, @Vitalius1 , this is affecting Azure DevOps.

**Comment by mikewheaton** [comment]
See [this StackOverflow question](https://stackoverflow.com/questions/8747689/safari-tabbing-focus-doesnt-work-in-5-1-2) for the most likely fix.

The (new?) default behavior in Safari is to _not_ have tab move between actionable elements. I'm not sure why some of our buttons receive focus while others don't. Given that the native `button` element will not receive tab focus without changing this setting, it seems out of scope for our components to do anything special here.
